upgrade-cluster.yml

```diff
@@ -90,7 +99,6 @@
     - { role: kubespray-defaults}
     - { role: upgrade/pre-upgrade, tags: pre-upgrade }
```

Comment:
We want to cordon before upgrading any components, actually

Reply:
i was trying to upgrade cluster and it was draining nodes before upgrade